go/vt/vtctl/workflow/lookup_vindex.go

```diff
@@ -77,13 +77,27 @@ func (lv *lookupVindex) prepareCreate(ctx context.Context, workflow, keyspace st
 		materializeQuery string
 	)
 
+	if specs == nil {
+		return nil, nil, nil, nil, vterrors.Errorf(vtrpcpb.Code_INVALID_ARGUMENT, "no vindex provided")
+	}
+	if len(specs.Vindexes) != 1 {
+		return nil, nil, nil, nil, vterrors.Errorf(vtrpcpb.Code_INVALID_ARGUMENT, "only one vindex must be specified")
+	}
+
+	vindexName := maps.Keys(specs.Vindexes)[0]
+	vindex := maps.Values(specs.Vindexes)[0]
+
 	// Validate input vindex.
-	vindex, vInfo, err := lv.validateAndGetVindex(specs)
+	vInfo, err := lv.validateAndGetVindexInfo(vindexName, vindex, specs.Tables)
 	if err != nil {
 		return nil, nil, nil, nil, err
 	}
 
-	vInfo.sourceTable, vInfo.sourceTableName, err = getSourceTable(specs, vInfo.targetTableName, vInfo.fromCols)
+	if len(specs.Tables) < 1 || len(specs.Tables) > 2 {
+		return nil, nil, nil, nil, fmt.Errorf("one or two tables must be specified")
+	}
```

Comment:
It's unclear to me why we're enforcing you can only specify 1 vindex but you can specify 2 tables.